.github/workflows/ci-build.yml

```diff
@@ -34,3 +34,9 @@ jobs:
       - name: unit test
         run: |
           python -m unittest discover -vb
+      - name: pytest
+        run: |
+          pip install -e ".[sparse,test]"
+          python -m pytest --cov=optimism -Wignore
+          # we can also add the flag -n auto for parallel testing
```

Comment:
Adding this block here makes the entire test suite run twice: once with `unittest` as the runner, and then with `pytest` as the runner. If we're switching to pytest, you should go head and remove the previous block.